mkchromecast/__init__.py

```diff
@@ -77,10 +77,10 @@
     Example:
 
     ffmpeg:
-        python mkchromecast.py --encoder-backend ffmpeg -c ogg -b 128
+        mkchromecast --encoder-backend ffmpeg -c ogg -b 128
```

Comment:
The same and for all the following changes...

Reply:
Indeed... This was just an existing patch in the current debian packaging which I upstreamed as part of this. Actually the change goes *python mkchromecast.py* -> *bin/mkchromecast* I could rework the old patch or just drop it  - it's not the important part here. 

In other words: the documentation needs an overhaul (hey, this is user info in __init__.py...), and  in this context this is probably the least problem. The more I think of it, tyhe proper solution for now would be to drop this patch. 

Thoughts?

Reply:
I never knew where was one supposed to add this type of documentation. If it is moved out of __init__.py, is another file needed to add all of this?

Reply:
This  is really a discussion which belongs to #174. The short story is that there already is too many files, so adding yet another one is probably not the solution. In the long run, the user info should IMHO be centralized e. g. in the manpage , the website or README.md , with links from other locations.

Being the person I am, I would actually go for the manpage. Your mileage may vary, though.